influxdb_client/client/write/retry.py

```diff
@@ -54,12 +76,26 @@ def get_backoff_time(self):
         if consecutive_errors_len < 0:
             return 0
 
-        # Full Jitter strategy
-        backoff_value = self.backoff_factor * (self.exponential_base ** consecutive_errors_len) * self._random()
-        return min(self.max_retry_delay, backoff_value)
+        delay_range = self.backoff_factor
+        i = 1
+        while i <= consecutive_errors_len:
+            i += 1
+            delay_range = delay_range * self.exponential_base
+            if delay_range > self.max_retry_delay:
+                break
+
+        delay = delay_range * self._random()
+        # at least min_retry_delay
+        delay = max(self.min_retry_delay, delay)
```

Comment:
everything below min_retry_delay is min_retry_delay, to have better random distribution, we could have:
```suggestion
        delay = self.min_retry_delay + (delay_range-self.min_retry_delay) * self._random()
        # at least min_retry_delay
        delay = max(self.min_retry_delay, delay)
```